src/main/resources/com/cloudbees/plugins/credentials/CredentialsStoreAction/newDomain.jelly

```diff
@@ -37,7 +37,8 @@
         <f:entry title="${%Description}" help="/plugin/credentials/help/domain/description.html">
           <f:textarea name="description"/>
         </f:entry>
-        <f:entry title="${%Specification}" help="/plugin/credentials/help/domain/specification.html">
+        <f:entry title="${%Specification}" help="/plugin/credentials/help/domain/specification.html"
```

Comment:
shouldn't this be handled in core? seems strange as its just consecutive entries?